src/main/java/org/broadinstitute/hellbender/engine/FeatureDataSource.java

```diff
@@ -136,6 +136,17 @@ public FeatureDataSource(final File featureFile) {
         this(featureFile, null);
     }
 
+    /**
+     * Creates a FeatureDataSource backed by the provided path. The data source will have an automatically
+     * generated name, and will look ahead the default number of bases ({@link #DEFAULT_QUERY_LOOKAHEAD_BASES})
+     * during queries that produce cache misses.
+     *
+     * @param featurePath path or URI to source of Features
+     */
+    public FeatureDataSource(final String featurePath) {
```

Comment:
I see the purpose of this constructor but it seems a little overzealous to add this easy constructor to only use it for one test utility method. you may disagree though so I will leave it up to you.

Reply:
The new constructor seems generally useful, as currently if you want to support non-local files you have to call the 4-arg constructor, which is a bit cumbersome.

Reply:
should we add a Path constructor to while we're here?

Reply:
Nah, let's just add the one we need for this PR. We'll likely refactor all these constructors anyway when we get around to https://github.com/broadinstitute/gatk/issues/958